src/libultra/voice/voicesetadconverter.c

```diff
@@ -1,3 +1,70 @@
-#include "global.h"
+/**
+ * File: voicesetadconverter.c
+ */
 
-#pragma GLOBAL_ASM("asm/non_matchings/boot/voicesetadconverter/__osVoiceSetADConverter.s")
+#include "ultra64/controller_voice.h"
+#include "io/controller.h"
+#include "functions.h"
+#include "variables.h"
+
+#define SWRITEFORMAT(p) ((__OSVoiceSWriteFormat*)(p))
```

Comment:
Oh I must have missed this (looking at it it is all the formats I added). The arg should probably be `ptr` not just `p`. That way it mirrors the `READFORMAT` in `controller.h`.